htdocs/adherents/class/adherent_type.class.php

```diff
@@ -668,14 +669,42 @@ public function getNomUrl($withpicto = 0, $maxlen = 0, $notooltip = 0)
         return $result;
     }
 
-    /**
-     *     getLibStatut
-     *
-     *     @return string     Return status of a type of member
-     */
-    public function getLibStatut()
-    {
-        return '';
+	/**
+	 *    Return label of status (activity, closed)
+	 *
+	 *    @param  	int		$mode       0=long label, 1=short label, 2=Picto + short label, 3=Picto, 4=Picto + long label, 5=Short label + Picto, 6=Long label + Picto
+	 *    @return   string     	   		Label of status
+	 */
+    public function getLibStatut($mode = 0)
+	{
+		return $this->LibStatut($this->status, $mode);
+	}
+
+	/**
```

Comment:
@ptibogxiv add here
`// phpcs:disable PEAR.NamingConventions.ValidFunctionName.ScopeNotCamelCaps`

Reply:
@frederic34 thanks !

Reply:
@ptibogxiv just for LibStatut wich is not camelCaps